chrome/content/scaffold/scaffold.js

```diff
@@ -871,8 +879,16 @@ var Scaffold = new function () {
 					_editors[tab].focus();
 				}, 10);
 				break;
+			default:
```

Comment:
What about the first tab, `metadata`? I was under the impression that both `metadata` and `browser` have their first focusable element focused after selection by default, and this kinda does just that, except after a small delay. I could break this up into two separate cases - would it be more readable that way?

For context, I don't think this part is strictly required - the main point with these changes is to not have the focus go away from tabs during keyboard navigation. If you switch to voiceover, click around, and then turn it off, the focus may not start going back to the fields after one or two tab selections. So this is just for consistency rather than anything else